### ISSUE TITLE
Listing all loadbalancer rules of all projects fails

### ISSUE THREAD

**Issue by remibergsma** [MissionCriticalCloud/cosmic#536]
```
🐵 > list loadbalancerrules listall=true projectid=-1
Error 530: None
{
  "cserrorcode": 9999,
  "errorcode": 530,
  "uuidList": []
}
```

```
2017-09-08 15:35:04.289 ERROR [c.c.a.ApiServer] (logid: 3b41de26) (ctx: b3d84f9a) Unhandled exception executing api command: queryAsyncJobResult
com.google.gson.JsonParseException: Expecting object found: "mnt"
        at com.google.gson.JsonObjectDeserializationVisitor.visitFieldUsingCustomHandler(JsonObjectDeserializationVisitor.java:100)
        at com.google.gson.ReflectingFieldNavigator.visitFieldsReflectively(ReflectingFieldNavigator.java:63)
        at com.google.gson.ObjectNavigator.accept(ObjectNavigator.java:120)
        at com.google.gson.JsonDeserializationContextDefault.fromJsonPrimitive(JsonDeserializationContextDefault.java:85)
        at com.google.gson.JsonDeserializationContextDefault.deserialize(JsonDeserializationContextDefault.java:56)
        at com.google.gson.Gson.fromJson(Gson.java:551)
        at com.google.gson.Gson.fromJson(Gson.java:498)
        at com.google.gson.Gson.fromJson(Gson.java:467)
        at com.google.gson.Gson.fromJson(Gson.java:417)
        at com.google.gson.Gson.fromJson(Gson.java:389)
        at com.cloud.api.ApiSerializerHelper.fromSerializedString(ApiSerializerHelper.java:53)
        at com.cloud.api.query.dao.AsyncJobJoinDaoImpl.newAsyncJobResponse(AsyncJobJoinDaoImpl.java:56)
        at sun.reflect.GeneratedMethodAccessor411.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
```

**Comment by remibergsma** [comment]
More issues:

```
:monkey_face: > list publicipaddresses listall=true projectid=-1
Error 530: None
{
  "cserrorcode": 9999,
  "errorcode": 530,
  "uuidList": []
}
:monkey_face: > list vpcs listall=true projectid=-1
Error 530: None
{
  "cserrorcode": 9999,
  "errorcode": 530,
  "uuidList": []
}
```

**Comment by sanderv32** [comment]
Fixed in PR #662
